pkg/pillar/cmd/loguploader/loguploader.go

```diff
@@ -55,6 +57,7 @@ var (
 	deviceNetworkStatus = &types.DeviceNetworkStatus{}
 	debug               bool
 	debugOverride       bool
+	backoffMode         bool
```

Comment:
Can you add some comments indicating what the true vs false mode means?
Perhaps "mode" is not very descriptive. Is it that exponential backoff is enabled or something different?